app/javascript/gobierto_data/webapp/components/Dataset.vue

```diff
@@ -377,28 +386,20 @@ export default {
       // save the query executed
       this.storeRecentQuery();
 
-      // wrap the result in an small number of records
-      let query = "";
-      if (this.currentQuery.includes("LIMIT")) {
-        query = this.currentQuery;
-      } else {
-        query = `SELECT * FROM (${this.currentQuery}) AS data_limited_results LIMIT 100 OFFSET 0`;
-      }
+      let query = this.currentQuery;
 
       const params = { sql: query };
```

Comment:
Since we don't need to to nothing to the `this.currentQuery`, just left `const params = { sql: this.currentQuery };` Save time and code!